### ISSUE TITLE
MAJOR redesign of the audible site

### ISSUE THREAD

**Issue by macr0dev** [macr0dev/Audiobooks.bundle#20]
They've changed everything, so it's all broken.  Going to have to re-write all of the page searches to get the data.  They're introduced a hidden 'toastMessage' that has everything in it.  Could help to use that instead of scrapping the content throughout the page.

**Comment by dethrophes** [comment]
I've made an initial attempt at fixing this.
I think I got most of it.

**Comment by macr0dev** [comment]
I took a quick look at what you came up with.  VERY interesting.  If I'm reading it write (I'm running on 3 hours of sleep after my wife had an emergency appendectomy this morning.) it looks like you're accommodating for both the new version of the site and the old version (I don't remember seeing the toastMessage in the .de site I looked at.)

Once I get caught up on some sleep I'll merge it into my local copy and do some thorough tests.  Thanks for jumping in and taking a look so quick.  I got thrown a big curve ball today!

**Comment by macr0dev** [comment]
Ok, I got your new stuff merged into mine.  I also added some stuff to clean up the synopsis since the new version includes html tags for bolding, italics, paragraphs, etc.  Also added a couple of lines to comma separate the narrators so they'll tag separately.  And comma/space separate the author if there's more than one.  If you want to go check those out.

Looks like we're completely back in business.
